cfme/services/myservice/ui.py

```diff
@@ -55,9 +55,12 @@ class MyServicesView(BaseLoggedInPage):
 
     @property
     def in_myservices(self):
+        # Adding workaround for 5.11.0.16.
+        # Its collecting ['Services', 'My Services', 'Services', 'Providers']
         return (
             self.logged_in_as_current_user and
-            self.navigation.currently_selected == ["Services", "My Services"])
+            self.navigation.currently_selected[0:2] == ["Services", "My Services"]
```

Comment:
Well this is quite minor, but you only need `self.navigation.currently_selected[:2]` there

Reply:
yup, but its ok in case of indexing